### PR TITLE
fix: refactor alert component to use sap-theming and align with global design

### PR DESCRIPTION
**Pull request by InnaAtanasova**
- adds theme designer variables for colors 
- updates paddings and spacing according to Fiori3 guidelines
- removes extra code

### CODE REVIEW

File: src/_settings.scss [hunk 1]
```diff
@@ -4,6 +4,8 @@
 // library namespace
 $fd-namespace: fd !default;
 
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/sass_variables.scss";
```
Comment (jbadan) [inline src/_settings.scss]: We don't need the scss variables. The foundation should be in place to simply use the css variables.

File: src/alert.scss [hunk 2]
```diff
@@ -10,41 +10,64 @@
...
+  @mixin fd-alert-close-btn-container {
+    position: absolute;
+    width: 2rem;
+    height: 1.625rem;
+    top: 0.375rem;
+    right: 0.125rem;
+  }
+
+  // SUCCESS
+  $fd-alert-background-color--success: var(--sapSuccessBackground, $sapSuccessBackground) !default;
```
Comment (jbadan) [inline src/alert.scss]: These can just be `var(--sapSuccessBackground)`, postcss will do the rest

File: src/_settings.scss (hunk 1 above)
Comment (InnaAtanasova) [reply src/_settings.scss]: Removed the import

File: src/alert.scss (hunk 2 above)
Comment (InnaAtanasova) [reply src/alert.scss]: Removed the sass variables

**Comment by jbadan** [comment]
@InnaAtanasova I know it's actually a feature, but for versioning we should label it a fix since it's not breaking.

**Pull request merged**